lua/wikis/commons/Widget/Match/Ticker/Container.lua

```diff
@@ -20,19 +22,65 @@ local FilterConfig = Lua.import('Module:FilterButtons/Config')
 ---@operator call(table): MatchTickerContainer
 local MatchTickerContainer = Class.new(Widget)
 MatchTickerContainer.defaultProps = {
-	upcomingTemplate = 'MainPageMatches/Upcoming',
-	completedTemplate = 'MainPageMatches/Completed',
+	matchTicker = {
+		module = 'MatchTicker/Custom',
+		fn = 'newMainPage',
+		args = {
+			upcomingLimit = 10,
+			ongoingLimit = 10,
+			recentLimit = 10,
+		}
```

Comment:
These seems unused

Reply:
Ah right, only the `participant` entrypoint did support them